armi/reactor/converters/blockConverters.py

```diff
@@ -354,7 +355,8 @@ def _addBlockRings(
             tempInput = tempHot = blockToAdd.getAverageTempInC()
 
         for ringNum in range(firstRing, firstRing + numRingsToAdd):
-            numFuelBlocksInRing = blockToAdd.location.getNumPosInRing(ringNum)
+            numFuelBlocksInRing = locations.numPositionsInRing(blockToAdd, ringNum)
```

Comment:
I thought this module was deleted. will read on...

Reply:
still holds some legacy stuff that was hard to remove. Determination of ring/pos stuff without access to a grid instance, mostly. I added a bunch of context the the module level docstring of locations.py. let me know of that helps